__tests__/app.test.js

```diff
@@ -311,3 +311,118 @@ describe("GET /api/users", () => {
       });
   });
 });
+
+describe("GET /api/articles?sort_by=...&order=...", () => {
+  it("should get all articles sorted by title", () => {
+    return request(app)
+      .get("/api/articles?sort_by=title")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toBeSortedBy("title", { descending: true });
+      });
+  });
+  it("should get all articles sorted by topic", () => {
+    return request(app)
+      .get("/api/articles?sort_by=topic")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toBeSortedBy("topic", { descending: true });
+      });
+  });
+  it("should get all articles sorted by author", () => {
+    return request(app)
+      .get("/api/articles?sort_by=author")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toBeSortedBy("author", { descending: true });
+      });
+  });
```

Comment:
I like the idea of being this robust, but as long as the description is clear that it can be sorted by any valid column you could also have done this in 1 test